src/pages/DetailMemoryPage.js

```diff
@@ -0,0 +1,68 @@
+import styled from 'styled-components/macro'
+import Button from '../components/Button'
+import PropTypes from 'prop-types'
+import MemoriesIcon from '../assets/images/icons/galery.png'
+
+DetailMemoryPage.propTypes = {
+  image: PropTypes.node,
+  onNavigate: PropTypes.func.isRequired,
+}
+
+export default function DetailMemoryPage({ image, onNavigate }) {
+  return (
+    <Wrapper>
+      <ImageDetail>
+        <Image src={image} alt="Memory 1" width="100%" />
+      </ImageDetail>
+      <nav>
+        <ButtonStyle onClick={onNavigate}>
+          <Icon src={MemoriesIcon} alt="" />
+          <span>Back to memories</span>
+        </ButtonStyle>
+      </nav>
+    </Wrapper>
+  )
+}
+
+const Wrapper = styled.section`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  height: 86.9vh;
```

Comment:
That's an oddly specific height. Please check if it fits your demands on different browser (I.e. device) sizes.

Reply:
Looks also good on chrome with different mobile-sizes.